### DISCUSSION TITLE
采用直接用<script>引入的方式，示例代码不能运行

### DISCUSSION THREAD

**Discussion by SpikeFJ** [didi/LogicFlow#1378] [Q&A]
现在很多框架的示例都是默认用户是采用构建的方式来运行的，

所以虽然声明了自己支持script方式，但是示例代码还是基于构建的，

但是还是有一部分人是希望能够原生script方式使用的。

希望能够完善下原生脚本引入执行的示例。

比方
```
import LogicFlow from "@logicflow/core";
import "@logicflow/core/dist/style/index.css";

const lf = new LogicFlow({
  container: document.querySelector("#app"),
  grid: true,
});
```

**Comment by wumail** [comment]
https://github.com/didi/LogicFlow/blob/master/packages/core/index.html 这个就是，更完整的就没了
